<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../浏览器下载内容/chat.png</normaloff>../../浏览器下载内容/chat.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTextEdit" name="editMsgWindow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:#ffffff;
	border-style:none;
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>40</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*
{
	background-color:#000000;
	border-style:none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="profile">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/cyx/Pictures/1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="moreFuncLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../浏览器下载内容/more.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:#e8e8e7;
	border-style:none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*
{
	background-color:#ffffff
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="seachLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../浏览器下载内容/放大镜.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addContactButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../浏览器下载内容/add.png</normaloff>../../浏览器下载内容/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>450</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:#f5f5f5;
	border-bottom:1px solid #e8e8e8;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="chattingUserName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>430</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:#ffffff;
	border-style:none;
    border-top:1px solid #e8e8e7;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../浏览器下载内容/Folder Icon.png</normaloff>../../浏览器下载内容/Folder Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="historyButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../浏览器下载内容/bubble chat 1.png</normaloff>../../浏览器下载内容/bubble chat 1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:#e8e8e8;
	border-style:none;
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>250</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{
	width:42px;
	background-color:#0ffffff;	
	border-style:none;
	margin:0;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="contactTab">
    <attribute name="icon">
     <iconset>
      <normaloff>../../浏览器下载内容/people.png</normaloff>../../浏览器下载内容/people.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QListWidget" name="contactList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>205</width>
       <height>460</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{
	background-color:#e8e7e7;
	border-style:none;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="msgList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>450</width>
     <height>340</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
	background-color:#f5f5f5;
	border-style:none;
}
QListWidget::Item{
	width:430px;
	height:100px;
	background-color:#66cccc;
	margin:5px;
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
